packages/store/addon/-private/system/model/internal-model.ts

```diff
@@ -654,7 +663,7 @@ export default class InternalModel {
   }
 
   getBelongsTo(key, options) {
-    let resource = this._recordData.getBelongsTo(key);
+    let resource = (this._recordData as DefaultRecordData).getBelongsTo(key);
```

Comment:
Is this cast safe? If so, how do we *know* it's safe?

Note that if it *is* safe, then the `!`s on line 684 become unnecessary.

Reply:
it is safe; however, `resource` doesn't seem to receive the knowledge that the return of `getBelongsTo` has `_relationships` even with this cast, which could be due to not all files being typed yet.

it is safe for similar reasons to the `recordData as RecordData` return type you commented on earlier, basically relationship support doesn't exist for RecordDatas that don't at least extend DefaultRecordData